### ISSUE TITLE
Google Chrome version 40.0.2214.91 issue

### ISSUE THREAD

**Issue by mzzm** [Julienh/Sharrre#239]
All my Chrome browsers automatically updated to version 40.0.2214.91 during the past two days, and I noticed some problems with certain plugins and one of them is sharrre.

The share does not respond immediately or not at all if you mouse over the share button. Please use the demo page as an example: http://sharrre.com/example2.html

It worked perfectly before the update and not anymore. But, all other browsers, ie Firefox, Safari, even IE8, that know of seem to be working fine.

Not sure if anyone has the same issue. Is there a fix for this or is this a Google development problem?

Thanks!

**Comment by ewwink** [comment]
work fine in my chrome Version 40.0.2214.111 m, try disable all extension.

**Comment by Mabdella** [comment]
@mzzm  Yeah check your extensions ... disabled "Avira Browser Safety" and it works fine now
Thanks a lot @ewwink
